symbulate/random_variables.py

```diff
@@ -254,10 +259,16 @@ def __init__(self, random_variable, condition_event):
         
     def draw(self):
         """
-            A function, that takes no arguments, explaining how to draw one 
-        outcome from the Probability Space described by the random variable.
+        A function that takes no arguments and returns a single set 
+            of values from the probability spaces defined in the conditional 
+            random variable that meet the conditional event. 
+        The number of values returned in the set depend on the number of 
```

Comment:
This comment is confusing and should be removed. We're not defining multiple probability spaces here; we're defining multiple random variables.